### PR TITLE
new error messages

### PR DESCRIPTION
**Pull request by kthye**

### CODE REVIEW

File: Source/Icebreaker/Controllers/MessagesController.cs [hunk 1]
```diff
@@ -1,5 +1,4 @@
-namespace Icebreaker
```
Comment (kthye) [inline Source/Icebreaker/Controllers/MessagesController.cs:1]: I reformatted the brackets in this file so you'll see a lot of changes

File: Source/Icebreaker/Controllers/MessagesController.cs [hunk 2]
```diff
@@ -136,71 +123,73 @@ private async Task HandleMessageActivity(ConnectorClient connectorClient, Activi
                     };
 
                     await connectorClient.Conversations.ReplyToActivityAsync(optInReply);
-                }
-                else
-                {
-                    if (isGroupChat)
-                    {
+                } else {
+                    if (isGroupChat) {
```
Comment (kthye) [inline Source/Icebreaker/Controllers/MessagesController.cs:127]: This is what you should read in this file

File: Source/Icebreaker/Controllers/MessagesController.cs (hunk 1 above)
Comment (judyliu98) [reply Source/Icebreaker/Controllers/MessagesController.cs:1]: great can't wait to review

**Review by judyliu98 — COMMENTED** [review]
LGTM

**Review by judyliu98 — APPROVED** [review]
LGTM

**Pull request merged**